.travis.yml

```diff
@@ -32,8 +33,13 @@ before_install:
       tar -zxf "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}_linux_amd64.tar.gz" -C "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/"
     fi
   - |
-    stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/stripe-mock > /dev/null &
+    stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/stripe-mock \
+      -http-port ${STRIPE_MOCK_PORT} \
+      -spec test/openapi/spec3.json \
+      -fixtures test/openapi/fixtures3.json \
+      > /dev/null &
```

Comment:
Just wanted to make sure, but I assume that this is an artifact of an older attempt? It seems to be running basically the same thing as `Stripe::StripeMock`.

Reply:
Ah, I saw your note about running differently in CI. Still not sure I 100% understand the expected workflow here, but I'll leave that in a comment below.